python-lib/googlesheets_common.py

```diff
@@ -56,3 +56,64 @@ def get_tab_ids(config):
         if legacy_tab_id:
             return [legacy_tab_id]
     return tabs_ids
+
+
+def get_unique_slugs(list_of_names):
```

Comment:
Could be done later: to avoid duplicate code, `get_unique_slugs` could re-use the newly introduced `get_unique_names`.
```python
def get_unique_slugs(list_of_names):
    from slugify import slugify
    return get_unique_names([slugify(name, separator="_", lowercase=False) for name in list_of_names])
```